org-code-javabuilder/app/src/main/java/org/code/javabuilder/JavaSourceFromString.java

```diff
@@ -0,0 +1,26 @@
+package org.code.javabuilder;
+
+import java.net.URI;
+import javax.tools.SimpleJavaFileObject;
+
+/** A file object used to represent source coming from a string. */
```

Comment:
(very minor) These early check-ins tend to have a way of establishing precedent so just pointing out some very minor things for your team to consider.

I prefer javadoc comments always being multi-line even when they're short.  It makes it easier for me to parse them out from the code and quickly get a sense of what a class does.

Reply:
our linter enforces a single-line comment takes up one line. Do you think it's preferable to split this text into two lines or just have the start/end /* */ on their own lines? We may need to update our linter to allow that. I don't have a strong opinion either way.

Reply:
I _think_ javadoc comments have different rules from regular comments are always allowed to be multiline?
https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line

I prefer the /** and */ to be on their own lines so this block would be:
```
/** 
 * A file object used to represent source coming from a string.
 */
```

Reply:
Linter details if it's helpful:
Style Guide: https://google.github.io/styleguide/javaguide.html
Details: https://github.com/sherter/google-java-format-gradle-plugin

Reply:
For now I'm going to leave as-is because we would have to change the linter to do the comment as above (it blocks check-in if I try a multi-line comment with one line of text).